pytorch_lightning/plugins/training_type/single_tpu.py

```diff
@@ -28,17 +23,22 @@
 
 class SingleTPUPlugin(SingleDevicePlugin):
 
-    def __init__(self, device: Union[torch.device, int]):
+    def __init__(self, device: int):
         if isinstance(device, int):
```

Comment:
is there a point to this `if` now that you removed `torch.device`?

Reply:
Updated!